dynawo/sources/Modeler/DataInterface/IIDM/DYNGeneratorInterfaceIIDM.cpp

```diff
@@ -258,4 +258,21 @@ GeneratorInterfaceIIDM::getID() const {
   return generatorIIDM_.id();
 }
 
+std::vector<GeneratorInterface::ReactiveCurvePoint> GeneratorInterfaceIIDM::getReactiveCurvesPoints() const {
+  std::vector<GeneratorInterface::ReactiveCurvePoint> ret;
+  if (generatorIIDM_.has_reactiveCapabilityCurve()) {
+    const IIDM::ReactiveCapabilityCurve& reactiveCurve = generatorIIDM_.reactiveCapabilityCurve();
+    for (IIDM::ReactiveCapabilityCurve::const_iterator it = reactiveCurve.begin(); it != reactiveCurve.end(); ++it) {
+      ReactiveCurvePoint point(it->p, it->qmin, it->qmax);
+      ret.push_back(point);
+    }
+  }
```

Comment:
The use of the new methods are not visible in this PR (I suppose they are needed by DynaFlow launcher).
Nevertheless, I was wondering if the method should throw an exception or at least write a log when the generator doesn't have any reactive capability curves. 

@rosiereflo @gautierbureau any advice on your side too?

Reply:
@marcochiaramello How are things handled on DFL side where there is no reactive capability curves?

Reply:
The reactive limits can be expressed in two ways in IIDM: using reactive capability curves or minMaxReactiveLimits.

Reply:
open question: should it be dynawo or DLF that is in charge of providing the correct points depending on the iidm?

Reply:
From DFL point of view, at first, when we are about to generate the diagrams, either we use the points if they exist or we use pmin/pmax/qmin/qmax. DFL shouldn't have to do more than that for me

Reply:
For me, it is a correct behaviour if the reactive capabilities array is empty. It can mean that we use minMaxReactiveLimits

Reply:
Fine then.